package.json

```diff
@@ -39,8 +39,8 @@
     ],
     "coverageThreshold": {
       "global": {
-        "lines": 78,
-        "statements": 78
+        "lines": 70,
```

Comment:
Why?

Reply:
Dropped some highly-covered code... So the remaining code is less covered :(

Reply:
Dropping `lib/stack-traces.ts` (unexpected, only used by Tokens) allows dialing back up to `77%`.